presto-redshift/src/main/java/io/prestosql/plugin/redshift/RedshiftClient.java

```diff
@@ -62,6 +72,76 @@ protected void renameTable(JdbcIdentity identity, String catalogName, String sch
         }
     }
 
+    static String generateCreateTableSql(String fullyQualifiedTableName, ImmutableList.Builder<String> columnList, Optional<String> distKey, Optional<String> compoundSortKeys, Optional<String> interleavedSortKeys)
+    {
+        checkArgument(!(compoundSortKeys.isPresent() && interleavedSortKeys.isPresent()), "Cannot set both a compound sortkey and an interleaved sortkey in a Redshift table");
```

Comment:
I wasn't sure of the best way to do error handling here. 

Do I validate all the ways the query could be malformed upfront (ie: two sortkeys, sortkey or distkey that does not exist, etc)? Or do I skip validation and let Redshift give back errors for malformed queries?